src/HLSMonitorService.ts

```diff
@@ -47,42 +45,118 @@ export class HLSMonitorService {
       exposeRoute: true,
     });
 
-    this.fastify.get("/status", async (request, reply) => {
-      if (!this.hlsMonitor) {
+    this.fastify.get("/monitor/:monitorId/status", async (request, reply) => {
+      if (!this.hlsMonitors.has(request.params.monitorId)) {
         reply.code(500).send({
           status: "error",
           message: "monitor not initialized",
         });
       }
-      const logs = await this.hlsMonitor.getErrors()
+    
+      const logs = await this.hlsMonitors.get(request.params.monitorId).getErrors()
       reply
         .code(200)
         .header('Content-Type', 'application/json; charset=utf-8')
         .send({ logs: logs });
     });
 
-    this.fastify.get("/clear-errors", async (request, reply) => {
-      if (!this.hlsMonitor) {
+    this.fastify.delete("/monitor/:monitorId/clear-errors", async (request, reply) => {
```

Comment:
I think it would be more restful to have:

`HTTP DELETE /monitor/:monitorId/status`

to clear errors.